src/Command/PullRequest/PullRequestMergeCommand.php

```diff
@@ -125,6 +128,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->ensureRemoteExists($sourceRemote, $sourceRepository, $output);
 
         try {
+            if (false === $prType = $this->getPrType($input, $output, $prType)) {
```

Comment:
I actually want to improve this in follow-up PR, but thats more related to handling fatal errors in general.